controller.go

```diff
@@ -372,6 +372,12 @@ func syncPV(pv *PV) {
 				}
 			} else {
 				// The PV must have been created with this ptr; leave it alone.
```

Comment:
Np, I'll remove it in a PR

On Wed, Apr 13, 2016 at 4:00 PM, Tim Hockin <notifications@github.com>
wrote:

> In controller.go
> <https://github.com/pmorie/pv-haxxz/pull/4#discussion_r59614893>:
>
> > @@ -372,6 +372,12 @@ func syncPV(pv *PV) {
> >  				}
> >  			} else {
> >  				// The PV must have been created with this ptr; leave it alone.
>
> Oops, we raced
> … <#m_-6381365950788708635_>
> On Wed, Apr 13, 2016 at 12:58 PM, Paul Morie <notifications@github.com>
> wrote: In controller.go <#4 (comment)
> <https://github.com/pmorie/pv-haxxz/pull/4#discussion_r59614622>>: > @@
> -372,6 +372,12 @@ func syncPV(pv *PV) { > } > } else { > // The PV must
> have been created with this ptr; leave it alone. Remove this line now?
> @jsafrane <https://github.com/jsafrane> <https://github.com/jsafrane> —
> You are receiving this because you were mentioned. Reply to this email
> directly or view it on GitHub <
> https://github.com/pmorie/pv-haxxz/pull/4/files/f8c2d4cc22beae7a53d3d5ffb6ee82825562c423#r59614622>
>
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/pmorie/pv-haxxz/pull/4/files/f8c2d4cc22beae7a53d3d5ffb6ee82825562c423#r59614893>
>